docs/misc/desktop_updates.md

```diff
@@ -0,0 +1,13 @@
+# Desktop Updates
```

Comment:
nitpick: docs/SUMMARY.md should be updated to make this file accessible from https://docs.trezor.io/trezor-suite